app/src/main/java/com/example/plantsapp/presentation/ui/tasks/adapter/PlantWithTasksViewHolder.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.plantsapp.presentation.ui.tasks.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.plantsapp.R
+import com.example.plantsapp.databinding.ItemPlantWithTasksBinding
+import com.example.plantsapp.domain.model.PlantWithTasks
+
```

Comment:
Leave here
//TODO Maybe add shared RecyclerViewPool

Probably we will not do it, but it is just for myself